internal/certmanager/test_files/util.go

```diff
@@ -19,13 +19,8 @@ package test
 
 import (
 	"math/rand"
-	"time"
 )
 
-func init() {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Was this dead code? (In addition to use a deprecated api)

Reply:
No, the func was called implicitly when the package was imported. Seeding random gen explicitly is no longer needed (since v1.20). It's done automatically under the hood when the program starts.

Reply:
ah ok. Thanks